modules/imgcodecs/include/opencv2/imgcodecs.hpp

```diff
@@ -213,41 +214,43 @@ compression parameters :
         return 0;
     }
 @endcode
- */
+@param filename Name of the file.
+@param img Image to be saved.
+@param params Format-specific save parameters encoded as pairs, see cv::ImwriteFlags
+paramId_1, paramValue_1, paramId_2, paramValue_2, ... .
+*/
 CV_EXPORTS_W bool imwrite( const String& filename, InputArray img,
               const std::vector<int>& params = std::vector<int>());
 
-/** @overload */
```

Comment:
i changed order
from
CV_EXPORTS_W Mat imdecode( InputArray buf, int flags ); // overload
CV_EXPORTS Mat imdecode( InputArray buf, int flags, Mat* dst); // has description
to
CV_EXPORTS Mat imdecode( InputArray buf, int flags, Mat* dst);  // has description
CV_EXPORTS_W Mat imdecode( InputArray buf, int flags ); // overload

i wonder is there a solid rule which one is overload.
what happen if i rearrange as:
CV_EXPORTS_W Mat imdecode( InputArray buf, int flags );  // has description
CV_EXPORTS Mat imdecode( InputArray buf, int flags, Mat* dst);  // overload